apps/docs/src/components/Calendar.tsx

```diff
@@ -0,0 +1,41 @@
+'use client'
+
+import FullCalendar from '@fullcalendar/react'
+import dayGridPlugin from '@fullcalendar/daygrid'
+import timeGridPlugin from '@fullcalendar/timegrid'
+import listPlugin from '@fullcalendar/list'
+import googleCalendarPlugin from '@fullcalendar/google-calendar'
+
+// cuHacking calendar
+const GOOGLE_CALENDAR_ID = 'fcdba3f354d4e01552e2495d743105bd9efce4e1076cda3c1ec1454d0f5faa73@group.calendar.google.com'
+
+// temporary, add as an env variable later
+// google calendar api key
+const API_KEY = ''
+
+export default function Calendar() {
```

Comment:
Best practice is to [avoid default exports](https://dev.to/phuocng/avoid-using-default-exports-a1c)